test/lit/exec/memory-copy.wat

```diff
@@ -0,0 +1,175 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --output=fuzz-exec and should not be edited.
+
+;; RUN: wasm-opt %s --enable-bulk-memory --fuzz-exec-before -q -o /dev/null 2>&1 | filecheck %s
```

Comment:
No need for this line, I think: `--fuzz-exec` on the next line will run both before and after the pass, print, and compare.
```suggestion
```

Reply:
Is there an internal comparison, or just filecheck? One difference is that the memory.copy traps in the original and the polyfill executes an unreachable in the lowered version. This results in a different printout.

Reply:
Nevermind, ignore what I just deleted; I think I have it right now.

Reply:
I don't think we need to filecheck the second execution.

For me, `--pass --fuzz-exec` not erroring is proof that a pass does not break execution, and is the main benefit of this test.

Reply:
Yeah; the test auto-update does actually include the second execution (and it has the different trap message), so that's also covered anyway for free.